modules/ROOT/pages/authentication.adoc

```diff
@@ -15,6 +15,17 @@ passwd:
         - ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDC5QFS...
 ----
 
+=== SSH key locations
+
+sshd uses a helper program to read public keys from files in a user's `~/.ssh/authorized_keys.d` directory. Key files are read in alphabetical order, ignoring dotfiles. The standard `~/.ssh/authorized_keys` file is read afterward, in the usual way. To debug the reading of `~/.ssh/authorized_keys.d`, manually run the helper program and inspect its output:
```

Comment:
Do you maybe want to linkify the [helper program](https://github.com/coreos/ssh-key-dir)?